.github/workflows/people.yml

```diff
@@ -71,34 +71,76 @@ jobs:
         with:
           node-version: '18.x'
           cache: 'yarn'
-          cache-dependency-path: src/frontend/app/desk/yarn.lock
+          cache-dependency-path: src/frontend/yarn.lock
       - name: Install dependencies
         run: yarn install --frozen-lockfile
       - name: Test Desk App
         env:
           CI: true
         run: yarn test
 
-  build-front-desk:
+  test-e2e:
     runs-on: ubuntu-latest
-    env:
-      NEXT_PUBLIC_API_URL: http://localhost:8071/api/v1.0/
-    defaults:
-      run:
-        working-directory: src/frontend/app/desk
+    
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        
+      - name: Set services env variables
+        run: |
+          cp env.d/development/common.dist env.d/development/common
+          cp env.d/development/postgresql.dist env.d/development/postgresql
+          cp env.d/development/kc_postgresql.dist env.d/development/kc_postgresql
+          cp env.d/development/crowdin.dist env.d/development/crowdin
```

Comment:
can't we run make command? Could we factorize this in the `makefile` with an appropriate make command, as the same logic is repeated in the bootstrap.